docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -0,0 +1,749 @@
+            </td>
+        </tr>
+        <tr>
+            <td>`autocomplete`</td>
+            <td>Boolean</td>
+            <td></td>
+            <td>
+                Whether autocomplete should be offered for the field. This is only allowed for string fields and
+                probably `generateIndex` should be true.
+            </td>
+        </tr>
+        <tr>
+            <td>`initiallyVisible`</td>
+            <td>Boolean</td>
+            <td></td>
+            <td>If true, the field is initially visible in the UI.</td>
+        </tr>
+        <tr>
+            <td>`hideOnSequenceDetailsPage`</td>
+            <td>Boolean</td>
+            <td></td>
+            <td>If true, hides the field on the sequence details page.</td>
+        </tr>
+        <tr>
+            <td>`perSegment`</td>
+            <td>Boolean</td>
+            <td></td>
+            <td>
+                Whether this is a metadata field that should exist for each segment. If so fields will be created as
+                `fieldName_A`, `fieldName_B` in the case of an organism with segments `A` and `B`.
+            </td>
+        </tr>
+        <tr>
+            <td>`customDisplay`</td>
+            <td>Object</td>
+            <td></td>
+            <td>Custom display of the field on the sequence details page</td>
+        </tr>
+        <tr>
+            <td>`customDisplay.type`</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`customDisplay.url`</td>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`rangeSearch`</td>
+            <td>Boolean</td>
+            <td></td>
+            <td>If true, enables range search for numeric fields.</td>
+        </tr>
+        <tr>
+            <td>`ontology_id`</td>
+            <td>String</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`definition`</td>
+            <td>String</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`guidance`</td>
+            <td>String</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`example`</td>
+            <td>String</td>
+            <td></td>
+            <td>Example value for the field.</td>
+        </tr>
+        <tr>
+            <td>`ingest`</td>
+            <td>String</td>
+            <td></td>
+            <td>Which NCBI field to map to this field</td>
+        </tr>
+        <tr>
+            <td>`preprocessing`</td>
+            <td>[PreprocessingFunction (type)](#preprocessingfunction-type)</td>
+            <td>Identity function on input field with same name</td>
+            <td>Preprocessing function to run on the field</td>
+        </tr>
+    </tbody>
+</table>
+
+### PreprocessingFunction (type)
+
+### Preprocessing (type)
```

Comment:
Our nextclade pipeline is described here: https://github.com/loculus-project/loculus/blob/main/docs/src/content/docs/for-administrators/existing-preprocessing-pipelines.md

Reply:
Thanks, I added a link to that page.